### ISSUE TITLE
Reassign shortcut warning dialog: cannot be used by keyboard users.

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#11686]
### Steps to reproduce

1. Go to the shortcuts page of preferences.
2. Select the about audacity action, and press Enter. An edit shortcut sequence dialog opens.
3. Press Ctrl + B for the new shortcut
4. Tab to the Save button and press it. A reassign shortcut warning dialog opens.
5. Press Tab.

### Expected behavior

The OK button becomes the focus

### Actual behavior

The import button in the preferences dialog becomes the focus.
For keyboard users it's not possible to use the reassign shortcut warning dialog.

### Audacity Version

Other

### Operating system

Windows

### Additional context

Tested using commit c5a20ac
